test/e2e/performanceprofile/functests/1_performance/hugepages.go

```diff
@@ -203,6 +204,18 @@ func getCentosPod(nodeName string) *corev1.Pod {
 							MountPath: "/dev/hugepages",
 						},
 					},
+					SecurityContext: &corev1.SecurityContext{
```

Comment:
I assume this is a fix for the security context that should land in another PR?

Reply:
yes, turns out we need fixes here: https://github.com/openshift/cluster-node-tuning-operator/pull/405